tests/test_generate_prsn.py

```diff
@@ -67,7 +67,7 @@ def test_create_prsn_netcdf_from_source(tiny_dataset, fake_dataset):
     assert np.shape(fake_dataset.variables['prsn']) == (11688, 4, 2)
     assert fake_dataset.variables['prsn'].standard_name == 'snowfall_flux'
     assert fake_dataset.variables['prsn'].long_name == 'Precipitation as Snow'
-
+    assert fake_dataset.variables['prsn'].units == 'g cm-2 s-1'
```

Comment:
There's not actually any check on the data values in this test, so it's hard to know whether it's doing the conversion correctly. Could you work out what the sum snowfall in this data *should* be an add that to the test?

Reply:
I believe the dataset used for `test_process_to_prsn` returns a prsn array of all zeros since these tests pass.
```python
result = fake_dataset.variables['prsn'][:] 
result = np.where(result != 0)
for array in result:
    assert len(array) == 0
```
Is there possibly another dataset I could use to test the conversion?

Reply:
Oh right, you had mentioned that. Well that's not a great testing dataset, then is it?! :)

Maybe you could throw a few strategic low temperature values into the test data set... it might throw some other tests out of whack, but you could adjust as necessary.

Reply:
Since we're only concerned about the conversion, would it be ok if I store the `pr` dataset in two variables, perform the conversion on one of them, and check to make sure that each value in the two datasets differs by a factor of 10? Or would you prefer I modify the data set and tests where needed?